package.json

```diff
@@ -98,6 +98,7 @@
     "terra-enzyme-intl": "^3.0.0",
     "terra-toolkit": "^6.2.0",
     "url-loader": "^4.1.1",
+    "uuidv4": "^3.0.0",
```

Comment:
The documentation says that `uuidv4` will be deprecated in the future and to use https://www.npmjs.com/package/uuid instead. Can you make this change?

Reply:
Fixed [here](https://github.com/cerner/terra-graphs/pull/117/commits/382664c0f4a6e411b79d006acf424f7431e5e7cd#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R101).